src/audio/pipeline.c

```diff
@@ -462,12 +462,19 @@ static void pipeline_comp_trigger_sched_comp(struct pipeline *p,
 static int pipeline_comp_trigger(struct comp_dev *current, void *data, int dir)
 {
 	struct pipeline_data *ppl_data = data;
+	int is_single_ppl = comp_is_single_pipeline(current, ppl_data->start);
+	int is_same_sched =
+		pipeline_is_same_sched_comp(current->pipeline,
+					    ppl_data->start->pipeline);
```

Comment:
Why don't you put pipeline_is_same_sched_comp on the same line with int is_same_sched? It does fit inside 80 char limit.

Reply:
@dbaluta But then `ppl_data->start->pipeline` can't be aligned with opening bracket, because it will exceed 80 chars.

Reply:
oh, ok. I see your point.